lte/gateway/c/oai/include/sctp_messages_types.h

```diff
@@ -89,6 +97,8 @@ typedef struct sctp_init_s {
   struct in6_addr ipv6_address[10];
   uint16_t port;
   uint32_t ppid;
+  uint32_t ngap_port;
+  uint32_t ngap_ppid;
```

Comment:
you don't need separate fields for ngap. A sctp connection is either s1ap or ngap, not both. You should simply put the right port and ppid numbers.